README.md

```diff
@@ -0,0 +1,37 @@
+# nomad-3rdparty - Set up Dependencies for Building Nomad
+
+[Homepage](https://github.com/mrosset/nomad)
+
+Nomad-3rdparty is a set of scripts to build and install dependencies for Nomad under Linux. This is partly due to g-golf not being included in Ubuntu apt packages. The entire Nomad project is under the GNU General Public License, version 3 or later.
+
+## System Requirements
+
+See the other [Readme](https://github.com/mrosset/nomad/tree/master/org) for ggeneral Nomad build instructions. Make sure when checking out this repository to use the --recursive flag to retrieve the submodules. Nomad is intended to work with Guile 2.
```

Comment:
can you link to http://git.savannah.nongnu.org/cgit/nomad.git/tree/?h=devel which is the main repo. github is just a mirror and covience for github users.